roles/tendrl-alerting/tasks/main.yml

```diff
@@ -2,11 +2,14 @@
 # installation
 #
 
-- include: source.yml
-  when: install_from == "source"
+# TODO Add when packages are ready
+#- name: Install tendrl-alerting
+#  yum:
+#    name=tendrl-alerting
+#    state=latest
 
-- include: binary.yml
-  when: install_from == "packages"
+- fail: msg="There are no packages yet, we can't start and enable it"
+  when: not ignore_incomplete_installation
```

Comment:
I guess that this is no longer true and `tendrl-alerting` package is available for some time. Which brings me to another detail: do we need this role when we don't seem to use it anymore?